documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,326 @@
+You can use the ***nether.analytics.gameStart*** method to send game start events.
+##### Example usage
+```javascript
+nether.analytics.gameStart();
+```
+The game session Id and the gamer tag are logged as parameters when a game start event is created.
+
+#### Game Stop
+You can use the ***nether.analytics.gameStop*** method to send game stop events.
+##### Example usage
+```javascript
+nether.analytics.gameStop();
+```
+The game session Id is logged as a parameter when a game stop event is created.
+
+#### Level Start
+You can use the ***nether.analytics.levelStart(level)*** method to send level start events.
+##### Example usage
+```javascript
+nether.analytics.levelStart(1);
+```
+***level*** The level Id or the level name.
+The game session Id is logged as a parameter when a level start event is created.
+
+#### Level Finished
+You can use the ***nether.analytics.levelFinished(level)*** method to send level finished events.
+##### Example usage
+```javascript
+nether.analytics.levelFinished(1);
+```
+***level*** The level Id or the level name.
+The game session Id is logged as a parameter when a level finished event is created.
+
+#### Custom event
+You can use the ***nether.analytics.customEven(event)*** method to create custom events.
+##### Example usage
+```javascript
+var event = {}
+event.name = 'example event';
+event.value = 10;
+event.id = 'de346ea2-4e59-49d8-863b-e0b2823b4ebd';
+nether.analytics.customEvent(event)
+```
+### Identity
+
+
+### Leaderboard
+
+#### Get Leaderboard
+You can use the ***nether.leaderboard.getAllLeaderboards(callback)*** method to get the currently configured Leaderboard.
+##### Example usage
+```javascript
+showLeaderboards = function(leaderboards) {
+    for (let leaderboard of leaderboards) {
+        var leaderboardName = leaderboard.name;
+        var leaderboardUrl = leaderboard._link;
+    }
+}
+nether.leaderboard.getAllLeaderboards(showLeaderboards);
+
+```
+### Get Leaderboard by name
+You can use the ***nether.leaderboard.getLeaderboard(leaderboardname)*** method to get all of the gamer tags and highscores for the specified Leaderboard.
+#### Example usage
+```javascript
+showLeaderboard = function(leaderboard) {
+    for (let entry of leaderboard.entries) {
+        var gamertag = entry.gamertag;
+        var isCurrentPlayer = entry.isCurrentPlayer;
+        var rank = entry.rank;
+        var score = entry.score;
+    }
+    var currentPlayerGamerTag = leaderboard.currentPlayer.gamertag;
+    var currentPlayerRank = leaderboard.currentPlayer.rank;
+    var currentPlayerScore = leaderboard.currentPlayer.score;
+}
+nether.leaderboard.getLeaderboard('default', showLeaderboard);
+
+```
+
+### Put score
+You can use the ***nether.score.addScore(score, callback)*** method to add a players score.
+#### Example usage
+```javascript
+scoreAdded = function(scoreAdded) {
+    if (scoreAdded === false) 
+        console.log('Unable to add score');
+}
+nether.scores.addScore(100, scoreAdded);
+```
+
```

Comment:
Let's keep it for now, but we should go throught and decide what methods should be exposed by the SDK. For example we don't want to allow a player to reset scores, etc. These features should be explicit to admin users.